core/core-spacemacs-buffer.el

```diff
@@ -334,6 +334,24 @@ WIDTH: current external width of the note's frame."
        (setq ,caption nil
              ,caption-length 0))))
 
+(defvar spacemacs-buffer-note-preview-lines 5
+  "If it's a positive integer, show the notes first number of lines.
+If nil, show the full note.")
+
+(defvar spacemacs-buffer--note-preview-nr-of-removed-lines nil
+  "Store the number of removed lines from the notes:
+Quick Help and Release Notes.")
+
+(defun spacemacs-buffer//note-remove-after-line (line)
```

Comment:
seems like this function always take spacemacs-buffer-note-preview-lines as its argument.

If so we can remove the argument.

Reply:
Done.